californium-core/src/main/java/org/eclipse/californium/core/network/stack/Block2BlockwiseStatus.java

```diff
@@ -349,6 +349,7 @@ public void onReadyToSend() {
 	 */
 	public final void completeOldTransfer(Exchange newExchange) {
 		Exchange oldExchange;
+
```

Comment:
Please, try to omit changing files only for formatting.

Reply:
I will remove this.